GitUI/CommandsDialogs/FormBrowse.cs

```diff
@@ -237,7 +237,7 @@ public FormBrowse(GitUICommands aCommands, string filter)
                 PuTTYToolStripMenuItem.Visible = false;
             }
 
-            RevisionGrid.SelectionChanged += RevisionGridSelectionChanged;
+            RevisionGrid.SelectionChanged += async (o, args) => await RevisionGridSelectionChangedAsync(o, args);
```

Comment:
❗️ Do not use `async void`

❗️ Asynchronous operations with a duration longer than is captured by the caller must be tracked to ensure they cannot cross a test boundary (I'll provide more details about this one after review is complete)